### PR TITLE
refactor: replace Nusi Ellipsis

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Replace Nusi Ellipsis and fixed tooltip position flashing bug.

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | Fixed tooltip position flashing bug.  |
| 🇨🇳 中文    |  修复了文字提示弹窗位置闪烁的bug。  |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.3

## Which issue(s) this PR fixes:

Fixes # https://erda-org.erda.cloud/erda/dop/projects/387/issues/bug?id=73872&issueFilter__urlQuery=eyJ0aXRsZSI6Ium8oOaghyIsInN0YXRlQmVsb25ncyI6WyJPUEVOIiwiV09SS0lORyIsIldPTlRGSVgiLCJSRU9QRU4iLCJSRVNPTFZFRCJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&iterationID=506&type=BUG

### CODE REVIEW

File: core/src/common/components/ellipsis.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      // 某些场景 ... 的宽度会被折叠，+1 来判断
+      if (scrollWidth > clientWidth + 1) {
+        setEnableTooltip(true);
+        setTooltip(true);
+      } else {
+        setEnableTooltip(false);
+        setTooltip(false);
+      }
+    }, TOOLTIP_MOUSE_ENTER_DELAY);
+  }, [setTooltip]);
```
Comment (daskyrk) [inline core/src/common/components/ellipsis.tsx:53]: Why just `setTooltip ` but not include `setEnableTooltip`?

File: core/src/common/components/ellipsis.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import React, { useState, useRef, useCallback } from 'react';
+import { Tooltip } from 'nusi';
+import { AbstractTooltipProps } from '../interface';
```
Comment (daskyrk) [inline core/src/common/components/ellipsis.tsx:16]: Where is this interface file?

Comment (hujiahao-hjh) [reply core/src/common/components/ellipsis.tsx:16]: It's the 'core/src/common/interface.d.ts'.

Comment (hujiahao-hjh) [reply core/src/common/components/ellipsis.tsx:16]: It's the 'core/src/common/interface.d.ts'.

**Pull request merged**